modules/openxr/openxr_api.cpp

```diff
@@ -291,7 +288,7 @@ bool OpenXRAPI::create_instance() {
 
 	// Create our OpenXR instance
 	XrApplicationInfo application_info{
-		"", // applicationName, we'll set this down below
+		"Godot Game Engine", // applicationName, if we're running a game we'll update this down below.
 		1, // applicationVersion, we don't currently have this
```

Comment:
Should the `applicationVersion` be also updated to match the `engineVersion` and overridden when we're running a game?

Reply:
Uhm, yeah I guess that makes sense as in this case the `engineVersion` and `applicationVersion` are the same, we're still lacking an applicationVersion for the game itself, there is one in the expert settings but not sure if that is accessible